owfs.js

```diff
@@ -27,46 +27,45 @@ module.exports = function(RED) {
         this.paths = n.paths;
 
         var node = this;
-        if (node.host && node.port) {
-            node.on("input", function(msg) {
-                var client = new owfs.Client(node.host, node.port);
-                var paths = node.paths;
-                if (msg.topic) {
-                    paths = [msg.topic];
-                }
+        node.on("input", function(msg) {
+            var host = msg.host  ||  node.host;
+            var port = msg.port  ||  node.port;
+            //node.log("owserver " + host + ":" + port);
```

Comment:
Oops, sorry.  The fact that I had left that in popped into my brain at 3 o'clock this morning for some unfathomable reason.